storage/database/api-cli/configuring-private-networks-using-api.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Configuring Private Networks on Database Instances using the API
+description: This page explains how to configure Private Networks on Database Instances using the API
+dates:
+  validation: 2021-11-16
+  posted: 2020-11-16
+---
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+- You have a [Managed Database](/storage/database/concepts/#managed-database/)
+- You have [generated an API key](/console/my-project/how-to/generate-api-key/)
+
+</Message>
+
+With [Private Networks](/network/vpc/quickstart/), you can configure your own Layer-2 network with IPv4 adresses. This feature allows you to enhance the security of your architecture, by isolating it from the Internet. 
+Scaleway Database Instances now support Private Networks as a Beta feature. This document will guide you through the steps to configure the Private Network feature on a Scaleway Database Instance through the API. For more information how to configure Private Networks for Database Instances from the Scaleway Console, follow [this guide]()
+
+## Creating a new Private Network for the Database Instance
+
+1. Click **Private Networks** in the **VPC** section of the side menu.
+2. Click the **Create a Private Network button**. The creation wizard displays.
+3. Complete the following steps in the wizard:
+  * Enter a **Name** for your Private Network, or leave the randomly-generated name in place. Optionally, you can also add tags to help you organize your Private Networks.
+  * Choose an **Availability Zone**, which is the geographical region where your Private Network will be deployed. 
+  <Message type="important">
+  
+  Make sure that your Database Instances are geolocated in the same Availability Zone as your Virtual Instances, to be able to add them to the Private Network.
+
+  </Message>
+4. Click **Create a Private Network** to finish. Your Private Network is created. You are taken back to the Private Networks dashboard, where your new Private Network is now displayed in the list.
```

Comment:
```suggestion
4. Click **Create a Private Network** to create your Private Network. You will be redirected to the Private Networks dashboard, where your new Private Network will be displayed in the list.
```

Reply:
4. Click **Create a Private Network**. You will be redirected to the Private Networks dashboard, where your new Private Network will be displayed in the list.